_topic_maps/_topic_map.yml

```diff
@@ -2412,7 +2412,7 @@ Topics:
     File: hcp-sizing-guidance
   - Name: Overriding resouce utilization measurements
     File: hcp-override-resource-util
-  - Name: Installing the hosted control plane command line interface
+  - Name: Installing the hosted control plane command-line interface
```

Comment:
I  couldn't find the directive for using the hyphen; the ISG indicates 'no hyphen'; just in case you don't have a particular reason not to use the hyphen.

See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c

Reply:
Per "CLI noun" from https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c
> In most cases, use command line unless you need to make specific reference to the command-line interface, as opposed to the graphical interface.

IIUC, for "command line" -- need no hyphen
for "command-line **interface**" -- need hyphen

Reply:
Double-confirming on the channel